Modules/CustomFields/Services/CustomFieldsService.php

```
@@ -90,16 +91,17 @@ public function getByTable($table)
      *
      * @originalFile CustomField.php
      */
-    public function save($id = null, $db_array = null)
+    public function save(Request $request = null, $id = null, $db_array = null)
     {
         if ($id) {
             // GetController the original record before saving
             $original_record = $this->getById($id);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$original_record'. (undefined)

(UnusedLocalVariable)